server/src/entities/asset-files.entity.ts

```diff
@@ -18,12 +18,15 @@ export class AssetFileEntity {
   id!: string;
 
   @Index('IDX_asset_files_assetId')
-  @Column()
+  @Column({ nullable: true, default: null })
```

Comment:
Is it a good idea to make this nullable?

Reply:
Sidecars are now discovered independently of files in library scans can are possibly orphans